webApps/apps.json

```diff
@@ -0,0 +1,82 @@
+{
+  "apps": [
+    {
+      "id": "web-extensions-draw-io",
+      "name": "Draw.io",
+      "subtitle": "View and edit draw.io diagram files (.drawio).",
+      "license": "AGPL-3.0",
+      "versions": [
+        {
+          "version": "0.1.0",
+          "url": "https://github.com/owncloud/web-extensions/releases/download/draw-io-v0.1.0/draw-io-0.1.0.zip"
+        }
+      ],
+      "authors": [
+        {
+          "name": "ownCloud GmbH",
+          "url": "https://owncloud.com"
+        }
+      ],
+      "tags": ["editor", "diagram"],
+      "resources": [
+        {
+          "name": "GitHub",
+          "icon": "",
+          "url": "https://github.com/owncloud/web-extensions/tree/main/packages/web-app-draw-io"
+        }
+      ]
+    },
+    {
+      "id": "web-extensions-external-sites",
+      "name": "External Sites",
+      "subtitle": "Add links to the app switcher (embedded or external).",
+      "license": "AGPL-3.0",
+      "versions": [
+        {
+          "version": "0.1.0",
+          "url": "https://github.com/owncloud/web-extensions/releases/download/external-sites-v0.1.0/external-sites-0.1.0.zip"
+        }
+      ],
+      "authors": [
+        {
+          "name": "ownCloud",
+          "url": "https://owncloud.com"
+        }
+      ],
+      "tags": ["links", "app-switcher"],
```

Comment:
Maybe the tag "navigation" would better fit then links ? Not sure about that, we shouldn't hang up on those, can adjust those later

Reply:
changed it 👍